src/components/character-count/index.md.njk

```diff
@@ -85,6 +85,8 @@ If a user tries to send their response with too many characters, you must show a
 
 The error message tells users what went wrong and how to fix it. The count message provides live feedback and updates as a user types.
 
+The input shows a red border only when the user tries sending too many characters. If the number of characters is within the limit, the input does not show this border even when there's been an error. We felt it might cause the user difficulty if the border disappeared once they started typing.
```

Comment:
Looks good thanks! Here's a minor suggestion for keep consistent, particularly in line 37.

Also, not sure how to release this alongside https://github.com/alphagov/govuk-design-system/pull/2181?

```suggestion
The input shows a red border only when the user tries to enter more than the character limit. If the number of characters is within the limit, the input does not show this border even when there's been an error. We felt it might cause the user difficulty if the border disappeared once they started typing.
```

Reply:
@calvin-lau-sig7 Very nice! Thanks